.github/workflows/tests_lgpumpi_cpp.yml

```
@@ -129,6 +129,7 @@ jobs:
         run: |
           source /etc/profile.d/modules.sh && module use /opt/modules/ && module load ${{ matrix.mpilib }}/cuda-${{ matrix.cuda_version_maj }}.${{ matrix.cuda_version_min }}
           export CUQUANTUM_SDK=$(python -c "import site; print( f'{site.getsitepackages()[0]}/cuquantum')")
+          export SCIPY_OPENBLAS=$(python -c "import scipy_openblas32; import os; print(os.path.dirname(scipy_openblas32.get_lib_dir()))")
```

`scipy-openblas32` or `scipy_openblas32`? could you double-check the consistency?

```suggestion
          export SCIPY_OPENBLAS=$(python -c "import scipy-openblas32; import os; print(os.path.dirname(scipy_openblas32.get_lib_dir()))")
```

---

Double checked. `scipy_openblas32` is the correct module.